packages/react-examples/.storybook/preview.js

```diff
@@ -172,8 +172,10 @@ function generateStoriesFromExamples(key, stories, req) {
     return;
   }
 
-  if (key.endsWith('.mdx')) {
-    // out out of the custom naming for mdx, use meta information
+  const isCollocatedStory = segments.includes('src');
+
+  if (key.endsWith('.mdx') || isCollocatedStory) {
```

Comment:
Let's say the title defined in collocated stories is authoritative and required.